pkg/build/controller/strategy/util.go

```diff
@@ -189,6 +195,12 @@ func makeVolume(volumeName, refName string, mode int32, fsType policy.FSType, vo
 				DefaultMode: &mode,
 			}
 		}
+	case policy.CSI:
+		if volumeSource != nil && volumeSource.CSI != nil {
+			vol.VolumeSource.CSI = volumeSource.CSI.DeepCopy()
```

Comment:
depending on where the discussion thread in https://github.com/openshift/origin/pull/26646#discussion_r763126357 lands, we may perhaps want to force the setting of the `refreshResource` attribute to 'false' for any shared resource CSI volumes.

Reply:
in our team meeting yesterday, we reached the consensus that at this time at least we were not going to force `refreshResources` to be set to false

Reply:
I concur - the build controller should be agnostic with respect to the specific CSI driver used to provide the volume.